test/Ref__test.re

```diff
@@ -0,0 +1,29 @@
+open Jest;
+open Expect;
+
+module Button_with_ref = {
+  [@react.component]
+  let make = (~children, ~ref) => {
+    <button ref role="FancyButton"> children </button>;
+  };
+};
+
+let getByRole = (role, container) => {
+  ReactTestingLibrary.getByRole(~matcher=`Str(role), container);
+};
+
+[@mel.get] external innerHTML: Dom.element => string = "innerHTML";
+
+describe("ref", () => {
+  test("can be passed as prop to a component", () => {
+    let domRef = React.createRef();
+    let container =
+      ReactTestingLibrary.render(
+        <Button_with_ref ref={ReactDOM.Ref.domRef(domRef)}>
+          {React.string("Click me")}
+        </Button_with_ref>,
+      );
+    let button = getByRole("FancyButton", container);
+    expect(button->innerHTML)->toBe("Click me");
```

Comment:
Shouldn't we test here something about the `ref`? This test just shows that `Button_with_ref` is rendering ok.

Reply:
Sure, added another assertion but didn't want to go to clicks and other events to keep the test simple and fast

Reply:
Thanks. (I think the expect in line 27 can be removed, because it doesn't test anything related to the ref)